src/runtime/vm/vm.cc

```diff
@@ -262,6 +262,12 @@ void VirtualMachine::InvokePacked(Index packed_index, const PackedFunc& func, In
   func.CallPacked(TVMArgs(values.data(), codes.data(), arity), &rv);
 }
 
+std::string DebugPrint(NDArray array) {
+  const PackedFunc* fprint = Registry::Get("relay._ndarray_repr");
+  ICHECK(fprint) << "unable to find printing function for constants";
+  return (*fprint)(array);
+}
+
```

Comment:
Should we make this part of the ReprPrinter? I've been using something very similar in the VM, but I feel like we should make it more general.

Reply:
Agree on this